src/alphabet.rs

```diff
@@ -23,3 +23,112 @@ pub const FLICKR: &[u8; 58] = b"123456789abcdefghijkmnopqrstuvwxyzABCDEFGHJKLMNP
 /// The default alphabet used if none is given. Currently is the
 /// [`BITCOIN`](constant.BITCOIN.html) alphabet.
 pub const DEFAULT: &[u8; 58] = BITCOIN;
+
+/// Prepared Alpabet for [`EncodeBuilder`](crate::encode::EncodeBuilder) and [`DecodeBuilder`](crate::decode::DecodeBuilder).
+#[derive(Clone, Copy)]
+#[allow(missing_debug_implementations)]
+#[allow(missing_docs)]
+pub struct Alphabet {
+    pub encode: [u8; 58],
+    pub decode: [u8; 128],
+}
+
+impl Alphabet {
+    /// Create prepared alphabet.
+    pub fn new(base: &[u8; 58]) -> Alphabet {
```

Comment:
I decided to play around with finding a way to make this `const fn` since if anyone is using a custom alphabet the ergonomics is really not nice. Manually expanding the loops and relying on checked indexing for the assertion appears to work ok: [playground](https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=1568b79ce6fcc8e06e78bf6a1c8d31a6).

Reply:
Thank you for example of workaround.